src/main/webapp/i18n/messages_pl.xml

```diff
@@ -1952,265 +2114,281 @@
-            <a href="http://dspace.org">http://dspace.org</a>.    Modyfikacje wprowadzone przez zespół
-            <a href="http://www.ceon.pl/">CeON</a> <a href="http://www.icm.edu.pl/">ICM UW</a>
-        </p>
-    </message>
-    <message key="xmlui.dri2xhtml.structural.contact-link">Kontakt z nami</message>
-    <message key="xmlui.dri2xhtml.structural.feedback-link">Wyślij uwagi</message>
+	<!--!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-    <message key="xmlui.dri2xhtml.structural.head-subtitle">Repozytorium Centrum Otwartej Nauki</message>
+		                  dri2xhtml
 
-    <message key="xmlui.dri2xhtml.structural.profile">Profil: </message>
-    <message key="xmlui.dri2xhtml.structural.logout">Wyloguj</message>
-    <message key="xmlui.dri2xhtml.structural.login">Zaloguj</message>
+		!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!-->
 
-    <message key="xmlui.dri2xhtml.structural.search">Szukaj w DSpace</message>
-    <message key="xmlui.dri2xhtml.structural.search-advanced">Szukanie zaawansowane</message>
-    <message key="xmlui.dri2xhtml.structural.search-in-community">W tym zbiorze</message>
-    <message key="xmlui.dri2xhtml.structural.search-in-collection">W tej kolekcji</message>
 
-    <message key="xmlui.dri2xhtml.structural.pagination-previous">Poprzednia strona</message>
-    <message key="xmlui.dri2xhtml.structural.pagination-info">Wyświetlanie pozycji {0}-{1} z {2}</message>
-    <message key="xmlui.dri2xhtml.structural.pagination-next">Następna strona</message>
 
-    <message key="xmlui.dri2xhtml.structural.link_cc">Creative Commons</message>
-    <message key="xmlui.dri2xhtml.structural.link_original_license">Oryginalna licencja</message>
 
 
-    <!-- DS-METS-1.0-MODS.xsl -->
-    <!-- DS-METS-1.0-DIM.xsl -->
-    <!-- DS-METS-1.0-QDC.xsl -->
-    <message key="xmlui.dri2xhtml.METS-1.0.no-preview">Podgląd niedostępny</message>
-    <message key="xmlui.dri2xhtml.METS-1.0.no-title">Bez tytułu</message>
-    <message key="xmlui.dri2xhtml.METS-1.0.no-author">Nieznany autor</message>
-    <message key="xmlui.dri2xhtml.METS-1.0.non-conformant">
+	<!-- structural.xsl -->
+	<message key="xmlui.dri2xhtml.structural.footer-promotional">
+        <a href="http://di.tamu.edu" id="ds-logo-link">
+			<span id="ds-footer-logo"> </span>
+		</a>
+		<p>
+			This website is using Manakin, a new front end for DSpace created by Texas A&amp;M University
+			Libraries. The interface can be extensively modified through Manakin Aspects and XSL based Themes.
+			For more information visit
+			<a href="http://di.tamu.edu">http://di.tamu.edu</a> and
+			<a href="http://dspace.org">http://dspace.org</a>
+		</p>
+	</message>
+	<message key="xmlui.dri2xhtml.structural.contact-link">Kontakt z nami</message>
+	<message key="xmlui.dri2xhtml.structural.feedback-link">Wyślij uwagi</message>
+
+	<message key="xmlui.dri2xhtml.structural.head-subtitle">Repozytorium Centrum Otwartej Nauki</message>
```

Comment:
This seems to be an institution specific header, maybe something more generic like Repozytorium DSpace would be more appropriate?